ekglib/kgiri/namespace.py

```diff
@@ -31,11 +33,47 @@ def set_ekg_namespace_with_base(key: str, base: str, suffix: str):
 def set_kgiri_base(ekg_kgiri_base: URIRef):
     global kgiri_base
 
-    kgiri_base = ekg_kgiri_base
+    kgiri_base =  complete_ns_iri_ending(ekg_kgiri_base)
 
     set_ekg_namespace('KGIRI_BASE', kgiri_base)
     set_ekg_namespace_with_base('KGIRI', kgiri_base, kgiri_id_suffix)
     set_ekg_namespace_with_base('KGGRAPH', kgiri_base, kgiri_graph_suffix)
 
+    # print(f"SET kgiri_base to '{kgiri_base}'")
+
+
+def set_kgiri_base_replace(ekg_kgiri_base_replace: URIRef):
+    global kgiri_base_replace
+
+    kgiri_base_replace =  complete_ns_iri_ending(ekg_kgiri_base_replace)
+
+    set_ekg_namespace('KGIRI_BASE_REPLACE', kgiri_base_replace)
+
+    # print(f"SET kgiri_base_replace to '{kgiri_base_replace}'")
```

Comment:
enable and set to log, or delete

Reply:
This logging is too much noise, was just for debug and at this level it does not give any value. Instead will move the logic to the start of the script, so it can log at the beginning what the replace strategy is.